test/parallel/test-buffer-alloc.js

```diff
@@ -312,12 +312,12 @@ assert.strictEqual('TWFu', (Buffer.from('Man')).toString('base64'));
   assert.strictEqual(quote, b.toString('ascii', 0, quote.length));
 
   // check that the base64 decoder ignores whitespace
-  const expectedWhite = expected.slice(0, 60) + ' \n' +
-                        expected.slice(60, 120) + ' \n' +
-                        expected.slice(120, 180) + ' \n' +
-                        expected.slice(180, 240) + ' \n' +
-                        expected.slice(240, 300) + '\n' +
-                        expected.slice(300, 360) + '\n';
+  const expectedWhite = `${expected.slice(0, 60)} \n${
+                           expected.slice(60, 120)} \n${
+                           expected.slice(120, 180)} \n${
+                           expected.slice(180, 240)} \n${
+                           expected.slice(240, 300)}\n${
+                           expected.slice(300, 360)}\n`;
```

Comment:
Definitely not a fan of multiline string literals. An array with a join might be an even better option.

```js
const expectedWrite = [
  expected.slice(0, 60),
  expected.slice(60, 120),
  expected.slice(120, 180),
  expected.slice(180, 240),
  expected.slice(240, 300),
  expected.slice(300, 360)
].join(' \n');
```